javascript.md

```diff
@@ -102,9 +113,17 @@ if(true) {
 Rust:
 
 ```rust
-// TODO
+let do_something = |some_argument: i32| {
```

Comment:
It's [an anonymous function expression](https://doc.rust-lang.org/nightly/reference.html#lambda-expressions), and the bars are the args. I can definitely clarify :)